osu.Game/Overlays/Notifications/ProgressNotification.cs

```diff
@@ -71,7 +71,7 @@ protected override void LoadComplete()
             base.LoadComplete();
 
             // we may have received changes before we were displayed.
-            updateState();
+            Scheduler.AddOnce(updateState);
```

Comment:
What's the purpose of this? It was added among a few other changes but I can't immediately see it. Probably deserves a comment.

Reply:
Just debouncing because we can (changed to conform with other similar changes I've been gradually applying elsewhere).

No real purpose, can revert if preferred.